<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/viewmag_4571.png</normaloff>icons/viewmag_4571.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="title">
       <string>Активная база вирусов</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnSettings">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/configure_4372.png</normaloff>icons/configure_4372.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tbBasePath">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>28</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOpenBase">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Открыть</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/folder_blue_9892.png</normaloff>icons/folder_blue_9892.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Выбор режима проверки</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Сканироавние имени файла</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbFullScanName">
               <property name="text">
                <string>Название послостью</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbSelectedScanName">
               <property name="text">
                <string>Содержится в названии</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Сканирование расширения файла</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbFullScanExeption">
               <property name="text">
                <string>Расширение полностью</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbSelectedScanExeption">
               <property name="text">
                <string>Содержиться в расширение</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Сканирование содержимого файла</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbScanFile">
               <property name="text">
                <string>Сканировать содержимое</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_12" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnOpenFolder">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string>Выбрать директорию</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/folder_orange_5296.png</normaloff>icons/folder_orange_5296.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tbFolderPath">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Сканирование</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Проверяемая директория:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbScanFolder">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Проверяемый файл:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbScanFile">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnStart">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string>Сканировать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStop">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string>Стоп</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_13" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>Результаты проверки</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QWidget" name="widget_14" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_15" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Просмотрено объектов:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbScanFiles">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_16" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Обранужено следов вирусов:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="ldDetectVirus">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_17" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listVirus"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Сканер</string>
    </property>
    <addaction name="menuStart"/>
    <addaction name="menuStop"/>
    <addaction name="separator"/>
    <addaction name="menuClose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="menuSettings"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuStart">
   <property name="text">
    <string>Сканировать</string>
   </property>
  </action>
  <action name="menuStop">
   <property name="text">
    <string>Остановить</string>
   </property>
  </action>
  <action name="menuClose">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="menuSettings">
   <property name="text">
    <string>Редактировать базу</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOpenBase</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDataBaseVirus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOpenFolder</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFolderForScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStart</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSettings</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>baseSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuStart</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>startScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuStop</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>baseSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>stopScan()</slot>
  <slot>startScan()</slot>
  <slot>openFolderForScan()</slot>
  <slot>openDataBaseVirus()</slot>
  <slot>baseSettings()</slot>
 </slots>
</ui>
